lib/dockerregistry/fixtures.go

```diff
@@ -14,6 +14,7 @@
 package dockerregistry
 
 import (
+	"github.com/uber/kraken/core"
```

Comment:
I don't think lib should depend on core.
In general lib functions should be independant.